include/openssl/err.h

```diff
@@ -99,6 +99,7 @@ typedef struct err_state_st {
 # define ERR_LIB_PROP            55
 # define ERR_LIB_CRMF            56
 # define ERR_LIB_PROV            57
+# define ERR_LIB_CMP             58
```

Comment:
if you are already at it, you can update the copyright year here as well.

Reply:
hmmm, I see this header file uses a HEADER_ERR_H without OSSL_ prefix....

Reply:
While also this is not strictly part of the CMP contribution,
we can of course do also these general adaptations, as well as adding a comment as suggested above after the final `#endif`. Ok?

Reply:
Yes.  I always like to avoid polluting global namespaces when possible.

Reply:
This general improvement has been moved to #8890.